waterlog/src/Components/BtnResolve.js

```diff
@@ -1,57 +1,63 @@
-import React, { Component } from 'react'
+import React, {
+  Component
+} from 'react'
 
- 
 const APIUri = 'http://api.iot.retrotest.co.za/api/monitors';
 class BtnResolve extends Component {
-   
-    monitorInfo = ["id","type","max_flow","location","status"];
-	faults = 0;
+
+  monitorInfo = ["id", "type", "max_flow", "location", "status"];
+  faults = 0;
 
   constructor(props) {
     super(props);
     this.state = {
-			data: [{"id":0,"type":"xxxx","max_flow":0.0,"long":0.0,"lat":0.0,"status":"xxx"}],
+      data: [{
+        "id": 0,
+        "type": "xxxx",
+        "max_flow": 0.0,
+        "long": 0.0,
+        "lat": 0.0,
+        "status": "xxx"
+      }],
       faultyMonitors: 0,
-      text:'Resolve Issue'
-		}; 
-	}
-		
+      text: 'Resolve Issue'
+    };
+  }
+
   componentDidMount() {
     fetch(APIUri)
       .then(response => response.json())
       .then(data => {
-				this.setState({ data });
-			})
+        this.setState({
+          data
```

Comment:
Help me understand how you are setting the state in your callback function

Reply:
set the data state to data response, fixed in latest commit.